conf/config-default.yaml

```diff
@@ -101,12 +101,15 @@ apisix:
   dns_resolver_valid: 30          # valid time for dns result 30 seconds
   resolver_timeout: 5             # resolver timeout
   ssl:
-    enable: true
+    enable: false                 # ssl is disabled by default
+                                  # enable it to use your own cert and key
     enable_http2: true
     listen_port: 9443
     # ssl_trusted_certificate: /path/to/ca-cert # Specifies a file path with trusted CA certificates in the PEM format
                                                 # used to verify the certificate when APISIX needs to do SSL/TLS handshaking
                                                 # with external services (e.g. etcd)
+    ssl_cert: ""
```

Comment:
Here is the path or content of the certificate? we need to add comments.

Reply:
Thank you! fixed